hack/gen-swagger-doc/run-gen-swagger-docs.sh

```diff
@@ -20,5 +20,4 @@ if [ "$#" -lt 1 ]; then
 fi
 OUTPUT=${2:-${PWD}}
 
-docker run -v ${OUTPUT}:/output gcr.io/google_containers/gen-swagger-docs:v1 https://raw.githubusercontent.com/kubernetes/kubernetes/master/api/swagger-spec/$1.json https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/api/$1/register.go
-
+docker run -v ${OUTPUT}:/output gcr.io/google_containers/gen-swagger-docs:v1.1 https://raw.githubusercontent.com/kubernetes/kubernetes/master/api/swagger-spec/$1.json https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/api/$1/register.go
```

Comment:
No.

I thought it's just a minor change so I named it v1.1. Is the version important?